content/articles/an-integration-of-djangos-bulk-model-editor.md

```diff
@@ -0,0 +1,139 @@
+---
+author_name: "Nik Nyby"
+author_url: http://ctl.columbia.edu/about/team/nyby/
+date: 2017-03-31
+draft: true
+lede: "We needed bulk editing in our project management tool, and this post outlines how we've taken advantage of some of Django's built-in functionality to achieve this."
+poster: ""
+poster_source: ""
+poster_sourceurl: ""
+tags: ["javascript", "django"]
+title: An integration of Django's bulk model editor
+topics:
+- Research and Development
+type: post
+---
+The Django project maintains a
+[web application](https://docs.djangoproject.com/en/1.10/ref/contrib/admin/) for
+managing models in your Django database. They've included bulk editing
+functionality for operating on multiple models at once, like this:
```

Comment:
wording's a little awkward here. on a first pass, it wasn't clear to me that you were talking about django admin without seeing the link destination